src/main/java/ugh/LegacyUghImplementation.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
```

Comment:
You must adjust this as the mentioned file 'GPL3-License.txt' did not exists on this repo nor is this file correct for this project as the project is more or less licensed under LGPL.

Reply:
I don’t like the idea to design my own license header. Would it be okay to use the GDZ header as it is in all other files?

Reply:
See my answer above.